examples/lighting-app/realtek/zephyr/README.md

```
@@ -75,9 +75,9 @@ sudo apt-get install git gcc g++ pkg-config libssl-dev libdbus-1-dev libglib2.0-
 
 -   To build the demo application:
 
-        ```bash
-        $ ./scripts/build/build_examples.py --target realtek-rtl87x2g-lighting build
-        ```
+          ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)